examples/tfengine/generated/org_foundation/groups/terraform.tfvars

```diff
@@ -0,0 +1,32 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+exists = true
+
+project_id = "example-devops"
+
+parent_type = "organization"
+
+parent_id = "12345678"
+
+billing_account = "000-000-000"
+
+
```

Comment:
any chance these empty lines can be removed?

Reply:
They're there because of the `hclField` usage. 
It can be removed by changing for something more verbose, but I guess it's minor

Reply:
I found a way to remove them. So it's fixed